### PR TITLE
fix for entry_type assignment

### PR DESCRIPTION
**Pull request by sanbrock**

### CODE REVIEW

File: src/pynxtools/nomad/parser.py [hunk 1]
```diff
@@ -483,7 +483,28 @@ def parse(
             archive.metadata = EntryMetadata()
 
         # Normalise experiment type
-        app_def = str(self.nx_root).split("(")[1].split(")")[0].split(",")[0]
+        app_defs = str(self.nx_root).split("(")[1].split(")")[0].split(",")
+        print(app_defs)
```
Comment (lukaspie) [inline src/pynxtools/nomad/parser.py:487]: Can we remove the print statement?

File: src/pynxtools/nomad/parser.py [hunk 2]
```diff
@@ -483,7 +483,28 @@ def parse(
...
+                    app if app != rename_nx_for_nomad("NXroot") else "Generic"
+                )
+            except AttributeError:
+                pass
+        if len(app_def_list) == 0:
+            app_def = "Experiment"
+        app_def = ", ".join(app_def_list) + (
+            " Experiment" if len(app_def_list) == 1 else "Experiments"
+        )
+        print(app_def)
```
Comment (lukaspie) [inline src/pynxtools/nomad/parser.py:507]: same here

**Review by lukaspie — APPROVED** [review]
Please remove print statements, other than that LGTM.

File: src/pynxtools/nomad/parser.py (hunk 2 above)
Comment (sanbrock) [reply src/pynxtools/nomad/parser.py]: done

**Pull request merged**